pkg/pillar/cmd/zedrouter/acl.go

```diff
@@ -244,11 +246,65 @@ func createACLConfiglet(ctx *zedrouterContext, aclArgs types.AppNetworkACLArgs,
 	if err != nil {
 		return rules, depend, err
 	}
+	clearUDPFlows(aclArgs, ACLs)
 	rules = append(rules, dropRules...)
 	rules, err = applyACLRules(aclArgs, rules)
 	return rules, depend, err
 }
 
+func clearUDPFlows(aclArgs types.AppNetworkACLArgs, ACLs []types.ACE) {
```

Comment:
Would be helpful to add a comment before this function saying that it looks if the ACLs have a UDP portmap rule, and if so it clears out any Conntrack state for that port number.

Reply:
Oops I reviewed the commits separately. See you fixed that in the second commit.
Squash the commits?

Reply:
will do